operations/finance.md

```diff
@@ -394,6 +400,16 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 * `Cheque`
 * `Other`
 
+#### Accounting item substate
+
+##### Payment substates
+
+* `Charged`
+* `Canceled`
+* `Pending`
+* `Failed`
+* `Verifying`
```

Comment:
I would reorder it to follow the cycle of the payment -> Pending, Verifying, Charged, Canceled, Failed